Character.cpp

```diff
@@ -85,24 +93,36 @@
     //Fight function - a pointer to the enemy is passed as an argument
     void Character::fight(Character* enemy) 
     {
-        //Variable to keep track of who's turn it is currently - 'my' in this case refers to the Character object that called the 'fight' method
-        bool myTurn = true;
+        //We initialize the countdown variables - these keep track of the cooldown until a hit
+        int attacker_hitCountdown=0;
+        int enemy_hitCountdown=0;
 
+
+        //Attacker hits the enemy first, then the other way around
+        enemy->sufferDamage(this);
+        this->sufferDamage(enemy); //the enemy hits us
+        
+        
+        //The fight keeps on going until somebody is dead
         while (!enemy->isDead() && !this->isDead()) 
         {
-            switch (myTurn)
-            {
-            case true: enemy->sufferDamage(this); //we hit the enemy
-                break;
-            case false: this->sufferDamage(enemy); //the enemy hits us
-                break;
+
+            if(attacker_hitCountdown >= getACD()){
+                enemy->sufferDamage(this);
+                attacker_hitCountdown = 0;
+            }else{
+                attacker_hitCountdown++;
             }
 
-            //We swap the state of 'myTurn'
-            myTurn = !myTurn;
+            if(enemy_hitCountdown >= enemy->getACD()){
+                this->sufferDamage(enemy);
+                enemy_hitCountdown = 0;
+            }else{
+                enemy_hitCountdown++;
+            }
 
         }
 
         //We announce the winner
         std::cout << (!isDead() ? getName() + " wins. Remaining HP: " + std::to_string(getHP()) :  enemy->getName() + " wins. Remaining HP: " + std::to_string(enemy->getHP()) ) << std::endl;
```

Comment:
Ezt a cout-ot szedd ki innen. A Character osztályodban ne legyen cout.

Reply:
(Más megoldást kell alkalmazni, nem elég simán Ctrl+X .)

Reply:
A cout ki lett takarítva a Character.cpp-ből, alternatív megoldás született. (github pages is be lett állítva, és a readme is frissítésre került)